templates/registry/markdown/attribute_namespace.md.j2

```diff
@@ -0,0 +1,40 @@
+{#- This template is rendered per top-level registry namespace. -#}
+{#- It consists of two variables: -#}
+{#- - id: The top-level namespace id. -#}
+{#- - groups: A sequence of all attribute groups under this namespace. -#}
+{#-           This includes deprecated groups. -#}
+{% import 'stability.j2' as stability %}
+{% import 'notes.j2' as notes %}
+{%- set my_file_name = ctx.id | lower | replace("_", "-") ~ ".md" -%}
+{{- template.set_file_name(my_file_name) -}}
+
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# {{ ctx.id | TitleCase }}
+
+{% for group in ctx.groups | sort(attribute="id") %}
+{%- set group_name = group.id | split_id | list | reject("eq", "registry") | join(" ") -%}
```

Comment:
would be nice to have a filter for it

Reply:
https://github.com/open-telemetry/weaver/issues/113